tests/image-stream/utils.spec.js

```diff
@@ -25,29 +25,44 @@ describe('ImageStream: Utils', function() {
 
   describe('.getArchiveMimeType()', function() {
 
-    it('should return application/x-bzip2 for a bz2 archive', function() {
+    it('should resolve application/x-bzip2 for a bz2 archive', function(done) {
       const file = path.join(DATA_PATH, 'bz2', 'raspberrypi.img.bz2');
-      m.chai.expect(utils.getArchiveMimeType(file)).to.equal('application/x-bzip2');
+      utils.getArchiveMimeType(file).then((type) => {
```

Comment:
I can't quite tell, but IMHO it looks like this change is 'extra' to just 'getting rid of read-chunk'. (i.e. you could have got rid of `read-chunk` without also switching this from `return` to `resolve`?)
If that's the case, could you perhaps mention it in the commit-message too?

Does the `getArchiveMimeType` docstring need updating too?